src/app/tests/integration/chip_im_responder.cpp

```diff
@@ -44,7 +44,8 @@ namespace {
 // The CommandHandler object
 chip::TransportMgr<chip::Transport::UDP> gTransportManager;
 chip::SecureSessionMgr gSessionManager;
-chip::SecurePairingUsingTestSecret gTestPairing;
+chip::SecurePairingUsingTestSecret gTestPairing(chip::Optional<chip::NodeId>::Value(chip::kUndefinedNodeId),
+                                                static_cast<uint16_t>(0), static_cast<uint16_t>(0));
```

Comment:
I would add comments for the params to know what they mean and consider a cosntant for the 0.

```cpp
constexpr uint16_t kTestOnlyKeyId = 0;

chip::SecurePairingUsingTestSecret gTestPairing(chip::Optional<chip::NodeId>::Value(chip::kUndefinedNodeId),
￼                                                kTestOnlyKeyId /* peerKeyId */, 
                                                kTestOnlyKeyId /* localKeyId */);
```